contracts/modules/cover/Cover.sol

```diff
@@ -164,7 +164,11 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
       allocationRequest.globalCapacityRatio = GLOBAL_CAPACITY_RATIO;
       allocationRequest.capacityReductionRatio = product.capacityReductionRatio;
       allocationRequest.rewardRatio = GLOBAL_REWARDS_RATIO;
-      allocationRequest.globalMinPrice = GLOBAL_MIN_PRICE_RATIO;
+      if (product.minPrice != 0) {
+        allocationRequest.productMinPrice = product.minPrice;
+      } else {
+        allocationRequest.productMinPrice = DEFAULT_MIN_PRICE_RATIO;
+      }
```

Comment:
(style) would this be more readable as ternary?

Reply:
It was a bit longer line so I didn't like it, but can change it if there is a style preference.

Reply:
its less than 120 columns so should be ok.

personally it looks cleaner to me as all the `allocationRequest` assignments are neatly together

<img width="412" alt="Screenshot 2024-12-13 at 16 54 35" src="https://github.com/user-attachments/assets/042774e8-5490-4143-9eac-1852adb02438" />

Reply:
side note, we should probably discuss as a team the [draft style guide document](https://www.notion.so/nxmcommunity/solidity-style-guide-DRAFT-a6712c5ede8b42408097027fb5eb6c10) that I wrote and decide what style we should follow as a team

Reply:
Agree then. Changed.